script/deploy-terraform

```diff
@@ -3,8 +3,6 @@
 # exit on error or if a variable is unbound
 set -eu
 
-export TF_VAR_docker_image=$DOCKER_TAG
```

Comment:
Not needed becasue of line 36 and 68 right and the whole output from `build_and_push`?

Reply:
Yeah, exactly 👍